fdbserver/DDRelocationQueue.actor.cpp

```diff
@@ -1408,17 +1421,18 @@ ACTOR Future<Void> dataDistributionRelocator(DDQueue* self,
 						} else {
 							destTeamSelect = TeamSelect::ANY;
 						}
-						auto req = GetTeamRequest(destTeamSelect,
-						                          PreferLowerDiskUtil::True,
-						                          TeamMustHaveShards::False,
-						                          PreferLowerReadUtil::True,
-						                          ForReadBalance(rd.reason == RelocateReason::REBALANCE_READ),
-						                          inflightPenalty,
-						                          rd.keys);
+						destTeamSelect.setForRelocateShard(ForRelocateShard::True);
+						state GetTeamRequest req =
+						    GetTeamRequest(destTeamSelect,
+						                   PreferLowerDiskUtil::True,
+						                   TeamMustHaveShards::False,
+						                   PreferLowerReadUtil::True,
+						                   ForReadBalance(rd.reason == RelocateReason::REBALANCE_READ),
+						                   inflightPenalty,
+						                   rd.keys);
```

Comment:
Aggregate trace, e.g., what is the percentage of relocator request resolved by returnToSourceTeam?